packages/frontend-2/lib/viewer/composables/ui.ts

```diff
@@ -339,3 +339,33 @@ export function useThreadUtilities() {
 
   return { closeAllThreads, open, isOpenThread }
 }
+
+export function useMeasurementUtilities() {
+  const state = useInjectedViewerState()
+
+  const measurementSettings = state.ui.measurements.value
```

Comment:
This isn't reactive, you're just storing the unreactive settings (the thing you take out of .value on a ref is not reactive, only the ref itself is reactive) into a basic JS const. It's always going to be the same value, even if `state.ui.measurements` actually changes.